Gems/EMotionFX/Code/MysticQt/Source/DialogStack.cpp

```diff
@@ -81,6 +79,14 @@ namespace MysticQt
     // get rid of all dialogs and their allocated memory
     void DialogStack::Clear()
     {
+        for (Dialog& dialog : m_dialogs)
+        {
+            if (dialog.m_dialogWidget)
+            {
+                dialog.m_dialogWidget->deleteLater();
+            }
```

Comment:
Would it make sense to do this in the destructor of `Dialog` so in here it only clears the vector?

Reply:
Tried that but found out that there are situations where the Dialog data gets copied and the dialog widget is expected to not be destructed in this case.